docs/src/guides/smartweave/warp/write-interactions.md

```diff
@@ -0,0 +1,70 @@
+# Warp WriteInteractions
+
+SmartWeave contracts contain a current state and the ability to modify that state via function calls, the way you call a function on a SmartWeave Contract is by creating a transaction known as a SmartWeave Action. This action contains the function name you would like to call and the input parameters required by the SmartWeave Contract function. `contract.writeInteraction` is a function that creates a SmartWeave Action.
+
+## Code
+
+```ts
+import { WarpFactory } from 'warp-contracts'
+
+const warp = WarpFactory.forMainnet()
+const STAMP_PROTOCOL = 'FMRHYgSijiUNBrFy-XqyNNXenHsCV0ThR4lGAPO4chA'
+
+async function doStamp() {
+  const result = await warp.contract(STAMP_PROTOCOL)
+    .connect('use_wallet')
+    .writeInteraction({
+      function: 'stamp',
+      timestamp: Date.now(),
+      transactionId: 'zQhANphTO0DOsaWXhExylUD5cBN3a6xWvfn5ZCpmCVY'
+    })
+  console.log(result)
+}
+```
+
+When calling writeInteraction, you need to pass your input parameters, these are the parameters the contract is expecting to receive.
+
+::: warning
+Since SmartWeave contracts are evaluated in a lazy flow, you do not know if your interaction ran successfully until you evaluate the contract to the current state. Use [Warp readState](./readstate.md) to access the contract and determine if the interaction was applied successfully.
+:::
+
+## Dry Write
+
+Dry write is the feature you can use to apply your interaction to the current state in a simulated interaction, the interaction does not run on the permaweb, it runs locally. This is an opportunity to verify your interaction will be successful before you apply your interaction. 
```

Comment:
Same as above. 
`..simulated interaction, the interaction does not...`

simulated interaction. The interaction does not...